mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/chargedialog/ChargeDialogFragment.java

```diff
@@ -130,6 +130,7 @@ public void saveNewCharge() {
         chargesPayload.setDateFormat("dd MMMM yyyy");
         chargesPayload.setChargeId(chargeId);
         initiateChargesCreation(chargesPayload);
+        getDialog().dismiss();
```

Comment:
No, This is not right place to dismiss the dialog, do it https://github.com/chaitanyaparashar/android-client/blob/b8cc8a94d2a6c40ef33eaa2309da3535b501f4df/mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/chargedialog/ChargeDialogFragment.java#L196